redshift_sqlalchemy/dialect.py

```diff
@@ -264,6 +527,167 @@ def create_connect_args(self, *args, **kwargs):
         default_args.update(cparams)
         return cargs, default_args
 
+    @util.memoized_property
+    def _isolation_lookup(self):
+        extensions = __import__('psycopg2.extensions').extensions
```

Comment:
This method is unchanged; I just moved it from a different spot so that underscore methods come after the public methods.

It's not clear to me why this method exists here. It looks functionally identical to the method it overrides: https://github.com/zzzeek/sqlalchemy/blob/d14a4b480c3b43885707e4a6e2466589971ff4d5/lib/sqlalchemy/dialects/postgresql/psycopg2.py#L592-L601

Should we simply remove this method and fall back to the psycopg2 default?